src/target/target_kind.cc

```diff
@@ -309,6 +309,7 @@ TVM_REGISTER_TARGET_KIND("c", kDLCPU)
     .add_attr_option<String>("march")
     .add_attr_option<Integer>("workspace-byte-alignment")
     .add_attr_option<Integer>("constants-byte-alignment")
+    .add_attr_option<Integer>("num-cores")
```

Comment:
what's this for?

Reply:
meta_schedule expects this attribute, see https://github.com/apache/tvm/blob/main/src/meta_schedule/utils.h#L384

Reply:
when the `ParallelizeVectorizeUnroll` schedule rule and the `MutateParallel` got eliminated, the `num-cores` attribute is no longer needed.